### ISSUE TITLE
FE: Implement final PACT Act Wizard URL

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#15669]
## Description
IA approved final URL for PACT Act Wizard: va.gov/pact-act-eligibility

Need to implement this. Per Randi: this will involve changes across: 
* content-build
* devops
* vets-website

Also need to remove old Results pages 1-2 (used for research) - or should we put that content into the placeholder file?

## Acceptance Criteria
- [x] PACT Act Wizard is reachable at /pact-act-eligibility

**Comment by jilladams** [comment]
@cindymerrill next week, we may pull this into sprint, to update the PACT URL. We don't want to disrupt you during analysis or readout, or while Danielle crystalizes recommendations, in case folks need to use the prototype during that process. We can negotiate timing -- just let us know when you'll be finished using the prototype.  (Or: if that starts to become a bigger picture delay, we could provide new URLs to link to from the prototype. Not urgent for now.)

**Comment by cindymerrill** [comment]
@jilladams I would prefer to have the coded prototype continue to work while I'm doing analysis and presenting the readout (so I can open it up as needed to demo). So if the URL needs to change sooner than I'm ready, it would be great to have a new URL to link to from Survey Monkey.

**Comment by jilladams** [comment]
Noting: Readout is scheduled for Monday 10/30 so pulling this into sprint and we can just get it done after the readout.

**Comment by randimays** [comment]
[devops PR](https://github.com/department-of-veterans-affairs/devops/pull/13670)
[content-build PR](https://github.com/department-of-veterans-affairs/content-build/pull/1773)
[vets-website PR](https://github.com/department-of-veterans-affairs/vets-website/pull/26548)

**Comment by cindymerrill** [comment]
Neither the new URL nor the old one are working now, @randimays , as of 4:15 pm PT Friday. I'll check again before I leave today...The new one is http://staging.va.gov/pact-act-eligibility/results-1-1

**Comment by jilladams** [comment]
Slack thread: https://dsva.slack.com/archives/C52CL1PKQ/p1699040063898139

Can wait til daily deploys on Monday to confirm this is all working as expected or if we need an OOB devops deploy (?) to unstick it for recording Fran's demo as well.

**Comment by randimays** [comment]
URL issue is resolved. The devops/revproxy merge was the issue. We needed to allow client-side routing via that PR to enable Cindy's link to work. I asked Platform to do a staging deploy for my commit. They did a dev and staging deploy; each took 10 minutes. The change will go to production on Wednesday which will not change the result.

Cindy's URL is working: https://staging.va.gov/pact-act-eligibility/results-1-1
The main PACT Act URL is working: https://staging.va.gov/pact-act-eligibility

Closing!
